config/insultibot/SampleUtterances.txt

```diff
@@ -4,4 +4,18 @@ INSULT to give me an insult
 BACKHANDED_COMPLIMENT to compliment me
 BACKHANDED_COMPLIMENT for a compliment
 BACKHANDED_COMPLIMENT to give me a compliment
-HELP for a little bit of help
\ No newline at end of file
+HELP for a little bit of help
+HELP how do I use you
+HELP how do I use it
```

Comment:
In general we should also have plural utterances in here. These are designed to be party gimmicks.
HELP how do we use you
HELP how do we use it
HELP how do I even